Rework largest-series-product documentation (#17)
kytrinyx/problem-specifications: exercises/largest-series-product/introduction.md

# Introduction

> You are part of a SIGINT team that has intercepted a series of encrypted communication signals from a potential terrorist group.

Reviewer: ```suggestion
You are part of a SIGINT team that has intercepted a series of encrypted communication signals from a group of bank robbers.
```

(just in case we end up triggering people who were near 9/11 or one of the other things internationally etc)

Author: Yeah, good call.

The signals contain a long sequence of digits.

> Your team needs to use various digital signal processing techniques to analyze the signals and identify any patterns that may indicate the planning of an attack.

Reviewer: ```suggestion
Your team needs to use various digital signal processing techniques to analyze the signals and identify any patterns that may indicate the planning of an heist.
```

Author: ```suggestion
Your team needs to use various digital signal processing techniques to analyze the signals and identify any patterns that may indicate the planning of a heist.
```
